sangbok/main.xml

```diff
@@ -1720,7 +1720,7 @@
 		  category="Wihn"
 		  melody="Marche Militaire">
 		<comment>
-			D sjunges av damer; H av herrar
+			A sjunges traditionsenligt av damer; B av herrar
```

Comment:
```suggestion
			Enligt tradition sjunges A av damer; B av herrar
```

Reply:
I feel like that makes singing only one part optional, but it's probably just how I read it? Apart from that I prefer this suggestion and will change to it if no one else has the same problem when reading it as I do.

Reply:
I was thinking the opposite. That only Damer sings traditionally and then Herrar with the original form.

Reply:
Reading it today I agree with you. I'll commit it!